server/testutils/factories/cas1SpaceBooking.ts

```diff
@@ -15,28 +16,31 @@ class Cas1SpaceBookingFactory extends Factory<Cas1SpaceBooking> {
     return this.params({
       actualArrivalDate: undefined,
       actualDepartureDate: undefined,
-      departureReason: undefined,
-      departureMoveOnCategory: undefined,
+      departure: undefined,
     })
   }
 
   current() {
     return this.params({
       actualDepartureDate: undefined,
-      departureReason: undefined,
-      departureMoveOnCategory: undefined,
+      departure: undefined,
     })
   }
 
   nonArrival() {
     return this.params({
       actualArrivalDate: undefined,
       actualDepartureDate: undefined,
-      departureReason: undefined,
-      departureMoveOnCategory: undefined,
+      departure: undefined,
       nonArrival: cas1SpaceBookingNonArrivalFactory.build(),
     })
   }
+
+  departedNonBreach() {
+    return this.params({
+      departure: cas1SpaceBookingDepartureFactory.build({ parentReason: undefined, moveOnCategory: undefined }),
+    })
+  }
```

Comment:
That would be neither 'breach' _nor 'Planned move-on'_ -- maybe simply `departed`? Would it matter much to the tests if some breach or move-on info was randomly added?